apps/yapms/src/lib/components/sidebar/sections/savedmaps/SavedMaps.svelte

```diff
@@ -0,0 +1,25 @@
+<script lang="ts">
+	import { PocketBaseStore } from '$lib/stores/PocketBase';
+	import SavedMap from './SavedMap.svelte';
+	import SaveMap from './SaveMap.svelte';
+
+	$: maps = $PocketBaseStore.collection('user_maps').getFullList();
+
+	function refreshMaps() {
+		maps = $PocketBaseStore.collection('user_maps').getFullList();
+	}
+</script>
+
+<div class="divider">Saved Maps</div>
+<div class="flex flex-col gap-2 p-4">
```

Comment:
Either this or the sidebar should be scrollable if you have enough maps where it will go off screen.